packages/manager/modules/pci/src/projects/project/billing/history/legacy/history-legacy.html

```diff
@@ -36,57 +36,83 @@
+                    data-translate="cpbhd_monthly_bill_description"
                     data-translate-values="$ctrl.getBillingDateInfo()"
                 ></span>
-                <i
+                <div
+                    class="h2 font-weight-bold mb-0 mt-2"
+                    data-ng-if="$ctrl.data.details.totals"
+                >
+                    <span
+                        data-ng-bind="$ctrl.data.details.totals.total.toFixed(2)"
+                    ></span
+                    >&nbsp;
+                    <span
+                        data-ng-bind="$ctrl.data.details.totals.currencySymbol"
+                    ></span>
+                </div>
+                <a
                     data-ng-if="$ctrl.currentMonthlyBillingBill"
-                    class="fa fa-file-pdf-o"
-                    title="{{ :: 'cpbhd_bill_pdf_tooltip' | translate }}"
-                ></i>
-            </a>
-            <a
-                data-ng-href="{{ $ctrl.billingUrl }}"
-                data-translate="cpbhd_bill_link"
-                target="_top"
-            ></a>
+                    data-ng-href="{{ $ctrl.currentMonthlyBillingBill.pdfUrl }}"
+                    target="_top"
+                >
+                    <span
+                        data-translate="cpbhd_bill_pdf_link"
+                        data-translate-values="$ctrl.getBillingDateInfo()"
+                    ></span>
+                    <i
+                        class="fa fa-file-pdf-o"
+                        title="{{ :: 'cpbhd_bill_pdf_tooltip' | translate }}"
+                    ></i>
+                </a>
+                <div>
+                    <a
+                        class="align-middle"
+                        data-ng-href="{{ $ctrl.billingUrl }}"
+                        target="_top"
+                    >
+                        <span data-translate="cpbhd_bill_link"></span>
+                        <i
+                            class="oui-icon oui-icon-arrow-right"
+                            aria-hidden="true"
+                        ></i>
+                    </a>
+                </div>
+            </div>
         </div>
         <div class="row">
             <div class="col-lg-6">
                 <oui-tile>
-                    <span
-                        class="oui-heading_4 oui-tile__title"
+                    <h4
+                        ng-if="!$ctrl.isSubsidiaryWithPostPaidUsageBilling"
                         data-translate="cpbhd_monthly_header"
                         data-translate-values="$ctrl.getBillingDateInfo()"
-                    ></span>
+                    ></h4>
+                    <h4
+                        ng-if="$ctrl.isSubsidiaryWithPostPaidUsageBilling"
+                        data-translate="cpbhd_monthly_post_paid_header"
+                        data-translate-values="$ctrl.getBillingDateInfo()"
+                    ></h4>
```

Comment:
You can keep only one `<h4>` block
```suggestion
                    <h4
                        data-translate="{{:: ($ctrl.isSubsidiaryWithPostPaidUsageBilling ? 'cpbhd_monthly_post_paid_header' : 'cpbhd_monthly_header') "
                        data-translate-values=":: $ctrl.getBillingDateInfo()"
                    ></h4>
```

Reply:
It's possible to right it this way, but I prefere to stay away from putting too much logic in the html template.

Reply:
I agree that's why I propose to add just one if in the `translate` property instead of 2 ifs in each  `<h4>` block. Except the translation part, it's the same code.

Before 
```
<h4
    ng-if="$ctrl.isSubsidiaryWithPostPaidUsageBilling"
    data-translate="cpbhd_monthly_post_paid_header"
    data-translate-values="$ctrl.getBillingDateInfo()"
></h4>
<h4
    ng-if="!$ctrl.isSubsidiaryWithPostPaidUsageBilling"
    data-translate="cpbhd_monthly_header"
    data-translate-values="$ctrl.getBillingDateInfo()"
></h4>
```
after 
```
<h4
      data-translate="{{:: ($ctrl.isSubsidiaryWithPostPaidUsageBilling ? 'cpbhd_monthly_post_paid_header' : 'cpbhd_monthly_header') "
      data-translate-values=":: $ctrl.getBillingDateInfo()"
  ></h4>
```